Lib/aifc.py

```diff
@@ -915,7 +915,10 @@ def open(f, mode=None):
     else:
         raise Error("mode must be 'r', 'rb', 'w', or 'wb'")
 
-openfp = open # B/W compatibility
+def openfp(f, mode=None):
```

Comment:
Please add a comment that `test_pyclbr.py` should be updated after removing `openfp`.

Reply:
Comment added. After this merges I'm going to create an issue in Roundup and assign it to myself so it gets formally tracked.